### DISCUSSION TITLE
How to get http status code in response?

### DISCUSSION THREAD

**Discussion by boli2017** [earthboundkid/requests#107] [Support] [answered]
err = requests.
		URL("url").
		Method(http.MethodGet).
		ToJSON(res).
		Fetch(ctx)

**Comment by earthboundkid** [comment, marked as answer]
What's the exact need here? By default, `requests.Builder` will return an error if the status code is not a success code. The error can be examined with `HasStatusErr` or using `errors.As` to cast to `ResponseError`. If you need to allow only a certain set of codes (not any 2xxx), you can use `CheckStatus`. If you want to record codes after the fact, there's `LogTransport`.
